ts/components/screens/ListItemComponent.tsx

```diff
@@ -74,11 +74,12 @@ export default class ListItemComponent extends React.Component<Props> {
   public render() {
     return (
       <ListItem
-        style={[styles.listItem, styles.flexRow, this.props.style]}
         onPress={this.props.onPress}
         onLongPress={this.props.onLongPress}
+        style={[styles.listItem, styles.flexRow, this.props.style]}
         first={this.props.isFirstItem}
         last={this.props.isLastItem || this.props.hideSeparator}
+        underlayColor={"white"}
```

Comment:
declare a const outside the class
const underlayColor = "transparent";
```suggestion
        underlayColor={underlayColor}
```

Reply:
ok